libs/platform/table/components/table-cell-header-popover/table-cell-header-popover.component.ts

```diff
@@ -189,4 +191,18 @@ export class TableCellHeaderPopoverComponent implements AfterViewInit {
             this._listComponent?._setCurrentActiveItemIndex(0);
         }
     }
+
```

Comment:
create enum for placement strings

Reply:
there's enum `ColumnAlign` that can be imported and used